cg/cli/set.py

```diff
@@ -149,7 +216,7 @@ def sample(context, sample_id, kwargs, skip_lims, yes):
         click.echo(f"Would change from {key}={old_value} to {key}={new_value} on {sample_obj}")
 
         if not (yes or click.confirm(CONFIRM)):
-            context.abort()
+            continue
```

Comment:
Is it intentional to remove the abort here?

Reply:
yes, lets skip ahead and try the next one. There is a confirm before changing anyway